spec/debug/blocks.cr

```diff
@@ -0,0 +1,12 @@
+def f(v)
+  yield(v)
+end
+
```

Comment:
This code is useless now. You can remove it as it do nothing due to `Array#each`